pkgs/by-name/lc/lcagent/package.nix

```
@@ -0,0 +1,45 @@
+{
+  fetchFromGitea,
+  lib,
+  stdenv,
+  flex,
+  bison,
+  librecast,
+  libsodium,
+  lcrq,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "lcagent";
+  version = "0.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "librecast";
+    repo = "lcagent";
+    tag = finalAttrs.version;
+    hash = "sha256-Kr3VQ56V+Neo4CrKX5AasuftXNNJCx4NnsPz1UBBCog=";
+  };
+
+  buildInputs = [
+    bison
+    flex
+    lcrq
+    librecast
+    libsodium
+  ];
+  installFlags = [ "PREFIX=$(out)" ];
+  doCheck = true;
+
+  meta = {
+    changelog = "https://codeberg.org/librecast/lcagent/src/tag/v${finalAttrs.version}/CHANGELOG.md";
+    description = "lcagent is the Librecast multicast agent.";
```

```suggestion
    description = "Librecast multicast agent";
```
please see https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes